lib/msf/core/exploit/mssql.rb

```diff
@@ -69,7 +69,7 @@ def initialize(info = {})
         Opt::RPORT(1433),
         OptString.new('USERNAME', [ false, 'The username to authenticate as', 'sa']),
         OptString.new('PASSWORD', [ false, 'The password for the specified username', '']),
-
+        OptBool.new('TDSENCRYPTION', [ true, 'Use TLS/SSL for TDS data "Force Encryption"', false]),
```

Comment:
on MSSQL Server 2012, it doesn't seem like it matters if I have this option set (it works either way). Is this needed for specific versions of MSSQL Server?